backend/constants.py

```diff
@@ -1,5 +1,7 @@
 from pathlib import Path
-
-UPLOAD_ROOT = Path(__file__).parent / "assets"
-IMG_DIR = UPLOAD_ROOT / "images"
-AUD_DIR = UPLOAD_ROOT / "audios"
\ No newline at end of file
+Prent_DIR = Path(__file__).parent
+ASSETS = Prent_DIR / "assets"
+IMAGES_DIR = ASSETS / "images"
+AUDIOS_DIR = ASSETS / "audios"
+IMAGES_PATH = "assets/images"
```

Comment:
what is the need of AIMAGES_DIR; sin't it same as IMAGES_PATH?

```suggestion
IMAGE_PATH = "assets/images"
```